docs/stdlib/abstract.rst

```diff
@@ -92,26 +91,26 @@ There are a number of abstract numeric types extending ``anyscalar``:
 
     :index: any anytype
 
-    Abstract base scalar type for
-    :eql:type:`anyint`, :eql:type:`anyfloat`, and :eql:type:`decimal`.
+    Represents an abstract base scalar type for
+    :eql:type:`anyint`, :eql:type:`anyfloat` and :eql:type:`decimal`.
 
 
 Abstract Range Types
 ====================
 
-There are some types that can be used to construct :ref:`ranges
-<ref_std_range>`. These scalar types are distinguished by the following
-abstract types:
+Like abstract types, you may also specify :ref:`ranges <ref_std_range>`.
+These scalar types are distinguished by the following:
```

Comment:
Hm, I've trouble parsing the new version. `Like abstract types, you may also...` is making me think that the types we discuss in this section are not "abstract" but they literally are.

`These scalar types are distinguished by the following:` -- "distinguished" works weirdly here. Also "these scalar types" looks misplaced, this section and the paragraph talks about abstract types (well, they're also scalar, but that's not the point here!)

All in all, I agree that the current wording can be improved, but I don't quite like the new suggested one.

Reply:
@1st1 I think changing L101 to `With abstract types, [...]` may be best to eliminate confusion from wording. Let me know what you think.

Good catch on L102, this is misplaced writing, I modified the original wording which could mean this may have been here for a bit? I'll think on this, maybe making a generic reference to this with `Types can be distinguished by the following:` is more appropriate -- will think on this.